### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by spear0fd3stiny** [khaosmaou/improved-gameplay#3]
This bug is related to the latest 3.6v release and it has to do with the existance of bomb/explosive vests on game modes that are't either bomb threat or hostage rescue.

I just finished playing a mission on Neon Tomb (vanilla map) and I found one civilian using a bomb vest, near the end of the mission. I've been playing with this version since it released both on vanilla and modded maps, and although I haven't played all of them, this was the first time I've encountered this bug. The game mode was "Barricated Suspects"

The civilian wearing an explosive vest was by the pools/jacuzzi l area, next to one of the bathrooms. I saw on the mod description that you had removed any explosive vests on regular modes, that's why it caught my eye. At first I thought it was just a visual bug so I shot the vest, and obviously it blew up and killed me.

**Expected behavior**
"Explosive vests are now only found on the Bomb Threat and Hostage Rescue layers." - There shouldn't be a bomb vest on this particular layer/game mode

**Screenshots**
Couldn't record the situation

**Desktop (please complete the following information):**
 - Mod Version: 3.6v
 - Game Version: 25143
 - Layer and Map (Example: "Barricaded Suspects on Gas": Barricated Suspects on Neon Tomb

**Additional context**
I'm not using any other A.I mod, all the other mods I have are cosmetic, so I think there's no conflict on that end

**Comment by khaosmaou** [comment]
I have seen this happen a few times myself lately in the new version of the game.

Certain variables sometimes just plaid don't work or inherit correctly sometimes. I'm trying to investigate what is causing this. Something has changed drastically as traps aren't working correctly either.

**Comment by spear0fd3stiny** [comment]
Yes, I was also wondering about traps because on all those runs I don't think I got any, specially on brisa cova and the hotel, both 2 layers known for their number of traps. I thought that I was just being "lucky", but this bomb vest thing was too easy to spot. Glad you're already looking onto it.

Cheers

**Comment by spear0fd3stiny** [comment]
Not good indeed...
It may sound like a dumb qustion but do you think VOID knows about this? As in, has someone reported this issues just like map modders reported the world genthing (where the A.I can't interact with doors)

**Comment by khaosmaou** [comment]
I just reported it to the RoN modding discord to see if others get the same problem. I tried using another AI mod, and when you load up the game content with either that mod or my mod in FModel, you get the same garbled output in all AILevelData.ini files. Very strange .

I have hidden my mod on NexusMods until it is either fixed or I find my own workaround.
